models/BaseAdobe.cfc

```diff
@@ -1043,9 +1043,9 @@ component accessors=true extends='cfconfig-services.models.BaseConfig' {
 			for (var eventGatewayConfiguration in eventGatewayConfigurations) {
 				var currentEventGatewayConfiguration = {
 					"DESCRIPTION" = eventGatewayConfiguration.description,
-					"KILLONTIMEOUT" = eventGatewayConfiguration.killontimeout,
+					"KILLONTIMEOUT" = javacast("boolean", eventGatewayConfiguration.killontimeout),
 					"CLASS" = eventGatewayConfiguration.class,
-					"STARTTIMEOUT" = eventGatewayConfiguration.starttimeout,
+					"STARTTIMEOUT" = javacast("int", eventGatewayConfiguration.starttimeout),
```

Comment:
If you look elsewhere in the file you'll see I'm forcing ints by doing

```js
expression+0
```

Reply:
thanks